.github/workflows/python-tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
             torch-version: "1.12.*"
             numpy-version-pin: "<2.0"
           - os: ubuntu-20.04
-            python-version: "3.9"
+            python-version: "3.10"
```

Comment:
@ceriottm could you please not force push back changes I undid?

Reply:
Don't know how this happened TBH. ohshitgit I guess